cgroups/systemd/apply_systemd.go

```diff
@@ -259,6 +259,7 @@ func (m *Manager) Apply(pid int) error {
 func (m *Manager) Destroy() error {
 	m.mu.Lock()
 	defer m.mu.Unlock()
+	theConn.StopUnit(getUnitName(m.Cgroups), "replace")
```

Comment:
I guess "replace" doesn't mean anything for StopUnit? Not sure why it even needs a mode :)